src/Stratis.Bitcoin.Features.Wallet/Models/RequestModels.cs

```diff
@@ -139,6 +140,15 @@ public class BuildTransactionRequest : TxFeeEstimateRequest
 
     public class SendTransactionRequest : RequestModel
     {
+        public SendTransactionRequest()
```

Comment:
What's the purpose of having this constructor defined like this?

Reply:
Not sure. 
I should check if constructor without parameters is actually used in the code. If it's not used we can delete it.

Reply:
even if it's used, it's created by default when doing `new SendTransactionRequest();` ;-)

Reply:
If the class already contains a constructor that requires any parameters we can't do 
new SendTransactionRequest(); without specifically creating a parameter-less constructor. 
So that's why I added it- in case somewhere new SendTransactionRequest() is still used.

Reply:
ah yes indeed.